smart-contracts/osmosis/contracts/cl-vault/src/helpers/getters.rs

```diff
@@ -43,7 +43,7 @@ pub fn get_value_wrt_asset0(
     Ok(total)
 }
 
-pub fn get_asset_ratios(
+pub fn _get_asset_ratios(
```

Comment:
why the `_`-prefix?

Reply:
Because we are not using it anymore, We are again splitting 50% 50% the funds to swap, so i can remove it if we don't want it.